release: v1.55.0 (#6023)
apollographql/router: .changesets/fix_tninesling_demand_control_variable_check.md

> ### Include request variables in demand control scoring ([PR #5995](https://github.com/apollographql/router/pull/5995))
> 
> Fix demand control scoring for queries which use variables.

Reviewer: ```suggestion
### Include request variables in demand control scoring ([PR #5995](https://github.com/apollographql/router/pull/5995))

Demand control scoring in the router now accounts for variables in queries.

```
